fac29/NisaInvest-TFB-FE
# created signup form

## Description

I've created a SignUpForm component and SignUp page. You can view this at `/signup`.

![image](https://github.com/user-attachments/assets/4e760d9d-9d75-4bf0-96da-6c0bddef00e6)

I tried to add a variable `defaultReason` to avoid hard-coding a default value, in case the reasons change in the future but had some type errors so have commented this out for now.

## Related Issue
Closes #27

## How Can This Be Tested?
Run `npm install` > `npm run dev` and go to `/signup` to see the form in action.

## Checklist:
